Feature/map cutout

TylerGrantSmith opened the pull request.

TylerGrantSmith commented:
I hope I did that right....

kevinykuo commented on analysis/map.R:
```diff
@@ -8,29 +8,58 @@ exposures_by_hasc <- risks_table_mapped %>%
 
 bins <- c(0, 10^3,10^4, 10^5, 10^6, Inf)
 pal <- colorBin("YlOrRd", domain = brazil$exposures, bins = bins)
+<<<<<<< HEAD
```
These `<<<<`, `====`, etc. should be deleted after you've selected what you want to keep. They're there to delineate diverged code from different branches.

TylerGrantSmith replied:
Woops, should be fixed now.  Thought I had deleted them.

kevinykuo commented:
OK now you'll want to run the article build script to see if it turns out fine, and commit the html so it can be deployed.

TylerGrantSmith commented:
Ok,  I will have to do that when I get back home.  My work has the SUSEP site blocked, so I don't have the data here to actually run the build.   Thanks for your help

TylerGrantSmith commented:
@kevinykuo I have updated and processed the article with the new leaflet code.  In addition I expanded the fig height for the leaflet so it would initialize with the full country displayed.  I also added the article_cache folder into the .gitignore.  Not sure if that was appropriate, or should I be maintaining my own separate .gitignore?

kevinykuo commented:
That gitignore should be there so you're fine. Could you `git rebase` so we don't have ~60 extra commits? Make a zip backup of the project dir just in case if you're doing this for the first time ;)
